### ISSUE TITLE
关于FGVC-Aircraft的准确率

### ISSUE THREAD

**Issue by jackymoo** [Ylexx/HBPASM#2]
我在FGVC-Aircraft数据集上试验了一下，最后的准确率只有88.3(论文中是91.3)，请问这是什么原因呢?我的txt文件是按类别号顺序排列的，这个会造成这样的误差吗?求大佬指点!

**Comment by Ylexx** [comment]
列表是需要shuffle的，另外一些细节参数稍微有些不一样，finetune的时候在car和aircraft上batch_size设为8会好一些
trainloader = torch.utils.data.DataLoader(trainset, batch_size=8,
                                          shuffle=True, num_workers=4)

**Comment by jackymoo** [comment]
恩恩，好的，我再试试。
